config/jobs/kubernetes/kops/kops-presubmits.yaml

```diff
@@ -350,6 +427,7 @@ presubmits:
 postsubmits:
   kubernetes/kops:
   - name: kops-postsubmit
+    cluster: eks-prow-build-cluster
```

Comment:
Can the new cluster upload to "UPLOAD_DEST=gs://k8s-release-dev/kops/ci" ?

I suggest doing this one in a different PR.